steps/household/review-household.tsx

```diff
@@ -0,0 +1,142 @@
+import { Table } from "../../components/Table";
+import { TextArea } from "../../components/TextArea";
+import { TextAreaDetails } from "../../components/TextAreaDetails";
+import ProcessStepDefinition from "../../helpers/ProcessStepDefinition";
+import ProcessDatabaseSchema from "../../storage/ProcessDatabaseSchema";
+import processRef from "../../storage/processRef";
+
+import PageSlugs, { urlObjectForSlug } from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const step: ProcessStepDefinition = {
+  title: PageTitles.Household,
+  heading: "Review household members",
+  step: {
+    slug: PageSlugs.Household,
+    nextSlug: PageSlugs.Rent,
+    Submit: makeSubmit({
+      url: urlObjectForSlug(PageSlugs.Rent),
+      value: "Save and continue"
+    }),
+    componentWrappers: [
+      ComponentWrapper.wrapStatic(
+        new StaticComponent({
+          key: "household-members-table",
+          Component: Table,
+          props: {
+            headings: ["Full name", "Relationship to tenant", "Date of birth"],
+            rows: [
+              ["John Smith", "Father", "10/04/1967"],
+              ["Steve Davis", "Uncle", "12/03/1978"]
+            ]
+          }
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "household-document-images",
+          Component: ImageInput,
+          props: {
+            label: "Take photos of any household change documents",
+            name: "household-document-images",
+            hintText: "You can take up to 3 different photos" as
+              | string
+              | null
+              | undefined,
+            maxCount: 3
+          },
+          defaultValue: [],
+          emptyValue: [] as string[],
+          databaseMap: new ComponentDatabaseMap<
+            ProcessDatabaseSchema,
+            "household"
+          >({
+            storeName: "household",
+            key: processRef,
+            property: ["documents", "images"]
+          })
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "housing-move-schemes-details",
+          // eslint-disable-next-line react/display-name
+          Component: (props): JSX.Element => {
+            return (
+              <Details summary="Housing move schemes: downsizing, overcrowding or aged over 60">
+                <Paragraph>Schemes cover:</Paragraph>
+                <List
+                  type={ListTypes.Bullet}
+                  items={[
+                    <>
+                      <Link href="/">Downsizing</Link>: A payment made for
+                      downsizing
+                    </>,
+                    <>
+                      <Link href="/">Overcrowding</Link>
+                    </>,
+                    <>
+                      <Link href="">Seaside and country</Link> if aged over 60
+                    </>
+                  ]}
+                ></List>
```

Comment:
```suggestion
                />
```

Prefer self-closing.